ocs_ci/ocs/ui/base_ui.py

```diff
@@ -195,6 +201,59 @@ def take_screenshot(self):
         self.driver.save_screenshot(filename)
         time.sleep(0.5)
 
+    def do_clear(self, locator, timeout=30):
+        """
+        Clear the existing text from UI
+
+        Args:
+            locator (tuple): (GUI element needs to operate on (str), type (By))
+            timeout (int): Looks for a web element until timeout (sec) occurs
+
+        """
+        wait = WebDriverWait(self.driver, timeout)
+        element = wait.until(ec.element_to_be_clickable((locator[1], locator[0])))
+        element.clear()
+
+    def wait_until_expected_text_is_found(self, locator, expected_text, timeout=60):
+        """
+        Method to wait for a expected text to appear on the UI (use of explicit wait type),
+        this method is helpful in working with elements which appear on completion of certain action and
+        ignores all the listed exceptions for the given timeout.
+
+        Args:
+            locator (tuple): (GUI element needs to operate on (str), type (By))
+            expected_text (str): Text which needs to be searched on UI
+            timeout (int): Looks for a web element repeatedly until timeout (sec) occurs
+        return:
+            bool: Returns True if the expected element text is found, False otherwise
+
+
+        """
+        wait = WebDriverWait(
+            self.driver,
+            timeout=timeout,
+            poll_frequency=1,
+            ignored_exceptions=[
+                NoSuchElementException,
+                ElementNotVisibleException,
+                StaleElementReferenceException,
+                TimeoutException,
```

Comment:
You have to take this one out if using it below

Reply:
removed